spec/harvesting/harvest_data_setup.rb

```diff
@@ -1,3 +1,32 @@
+require 'rspec/core'
+module RSpec::Core::MemoizedHelpers::ClassMethods
+  def cassette_let!(name, &block)
+    let!(name, &block)
+
+    # partially duplicates the implementation found at:
+    # https://github.com/rspec/rspec-core/blob/58f38210492cd369784d3fe1a849c0e81342a2f2/lib/rspec/core/memoized_helpers.rb
+    if block.arity == 1
+      define_method(name) {
+        __memoized.fetch_or_store(name) {
+          VCR.insert_cassette("harvest_data_setup/#{name}", :record => :once, :allow_playback_repeats => true)
+          result = super(RSpec.current_example, &nil)
+          VCR.eject_cassette
+          result
+        }
+      }
+    else
```

Comment:
@mscottford If I understand this correctly, `block.arity` will always be 0 in this PR. Do you think it's worth to keep the path for `block.arity == 1`? Also, could you add documentation to this method? I don't fully understand what it does...

Reply:
Sure thing. I removed the support for `arity == 1` and I added some documentation to describe what the method does.